docs/installation/install_epinio.md

```diff
@@ -126,75 +140,85 @@ Ingress controller.
+- [Install on RKE2](other_inst_scenarios/install_epinio_on_rke.md) — Install Epinio on Rancher RKE2 cluster
+- [Install on K3d](other_inst_scenarios/install_epinio_on_k3d.md) — Install Epinio on K3d cluster
+- [Install on K3s](other_inst_scenarios/install_epinio_on_k3s.md) — Install Epinio on K3s cluster
+- [Install on Rancher Desktop](other_inst_scenarios/install_epinio_on_rancher_desktop.md) — Install Epinio on Rancher Desktop
+- [Install on EKS](other_inst_scenarios/install_epinio_on_eks.md) — Install Epinio on AWS EKS cluster
 
-> *NOTE*: The Public Cloud How-to lists the three major Cloud providers but Epinio can run on any Kubernetes cluster.
+:::note
+The Public Cloud How-to describes the three major cloud providers but Epinio can run on any Kubernetes cluster.
+:::
 
 ## Internal Epinio components
+
 ### Kubed
 
 Kubed is installed as a subchart when `.Values.kubed.enabled` is `true` (default).
-If you already have kubed running, you can skip the installation by setting
+If you already have kubed, you can skip installation by setting
 the helm value `.Values.kubed.enabled` to `false`.
 
 ### S3 storage
 
-Epinio is using an S3 compatible storage to store the application source code.
+Epinio uses an S3 compatible storage to store the application source code.
 This chart will install [Minio](https://min.io/) when `.Values.minio.enabled` is
 `true` (default).
 
-In addition to Minio, Epinio offers [s3gw](https://s3gw.io/) as another internal S3 compatible storage. It is installed when `.Values.minio.enabled` is set to `false` and `.Values.s3gw.enabled` is set to `true`.
+In addition to Minio, Epinio offers [s3gw](https://s3gw.io/) as another S3 compatible store.
+It is installed when `.Values.minio.enabled` is set to `false` and `.Values.s3gw.enabled` is set to `true`.
 
-Both choices for internal S3 compatible storage can be configured to use a user-defined storageClass. If no StorageClass is defined, the default storageClass is used. For Minio the custom storageClass is set via the value of `.Values.persistance.storageClass`. For s3gw the custom storageClass is set via the value of `.Values.s3gw.storageClass.name`.
+Both choices for internal S3 compatible storage can be configured to use a user-defined storageClass.
+If no StorageClass is defined, the default storageClass is used.
+Using Minio set the custom storageClass by the value of `.Values.persistance.storageClass`.
+Using s3gw set the custom storageClass by the value of `.Values.s3gw.storageClass.name`.
 
-Any external S3 compatible solution can be used instead by setting `.Values.minio.enabled` value to `false` (`.Values.s3gw.enabled` is `false` by default) and using [the values under `s3`](https://github.com/epinio/helm-charts/blob/b389a4875af9f03b484a911c49a14f834ba04b64/chart/epinio/values.yaml#L44) to point to the desired S3 server.
+<!--TODO: Not sure about using a hashed blob, is that OK? Presumably to guarantee a line number reference?-->
```

Comment:
Yes. A non-hashed reference, i.e. by branch name can shift what it points to, as the file changes on the branch.